server.tf

```diff
@@ -0,0 +1,347 @@
+
+resource "google_compute_firewall" "forseti-server-ssh-external" {
+  name                    = "forseti-server-ssh-external-${local.random_hash}"
+  project                 = "${var.project_id}"
+  network                 = "${var.vpc_host_network}"
+  target_service_accounts = ["${google_service_account.forseti_server.email}"]
+  source_ranges           = ["0.0.0.0/0"]
+  priority                = "0"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["22"]
+  }
+}
+
+resource "google_compute_firewall" "forseti-server-allow-grpc" {
+  name                    = "forseti-server-allow-grpc-${local.random_hash}"
+  project                 = "${var.project_id}"
+  network                 = "${var.vpc_host_network}"
+  target_service_accounts = ["${google_service_account.forseti_server.email}"]
+  source_ranges           = ["10.128.0.0/9"]
+  priority                = "0"
+
+  allow {
+    protocol = "tcp"
+    ports    = ["50051"]
+  }
+}
+
+#------------------------#
+# Forseti Storage bucket #
+#------------------------#
+resource "google_storage_bucket" "config" {
+  name     = "forseti-server-${local.random_hash}"
+  location = "${var.storage_bucket_location}"
+  project  = "${var.project_id}"
+}
+
+resource "google_storage_bucket_object" "forseti_server_config" {
+  name    = "configs/forseti_conf_server.yaml"
+  bucket  = "${google_storage_bucket.config.name}"
+  content = "${data.template_file.forseti_server_config.rendered}"
+}
+
+#-------------------------#
+# Forseti server instance #
+#-------------------------#
+resource "google_compute_instance" "forseti-server" {
+  name                      = "${local.server_name}"
+  zone                      = "${local.server_zone}"
+  project                   = "${var.project_id}"
+  machine_type              = "${var.server_type}"
+  allow_stopping_for_update = true
+
+  boot_disk {
+    initialize_params {
+      image = "${var.server_boot_image}"
+    }
+  }
+
+  network_interface {
+    network = "${var.vpc_host_network}"
+
+    # subnetwork = "projects/${var.vpc_project_id}/regions/$region/subnetworks/$subnet_id
+
+    access_config {}
+  }
+
+  metadata_startup_script = "${data.template_file.forseti_server_startup_script.rendered}"
+
+  service_account {
+    email  = "${google_service_account.forseti_server.email}"
+    scopes = ["cloud-platform"]
+  }
+
+  labels {
+    goog-dm = "${local.server_name}"
+  }
+
+  depends_on = [
+    "google_service_account.forseti_server",
+    "google_project_service.activate_services",
+  ]
+}
+
+#----------------------#
+# Forseti SQL database #
+#----------------------#
+resource "google_sql_database_instance" "master" {
+  name             = "${local.cloudsql_name}"
+  project          = "${var.project_id}"
+  region           = "${var.cloudsql_region}"
+  database_version = "MYSQL_5_7"
+
+  settings = {
+    tier              = "${var.cloudsql_type}"
+    activation_policy = "ALWAYS"
+    disk_size         = "25"
+    disk_type         = "PD_SSD"
```

Comment:
For users who are trying to keep the cost low, the ability to configure disk_type may be appreciated.

Reply:
Moved to #16